common/src/main/java/com/flipkart/flux/persistence/CryptHashGenerator.java

```diff
@@ -0,0 +1,22 @@
+package com.flipkart.flux.persistence;
```

Comment:
Add copyright comments, you can setup Intellij idea to to include copyright comments by default for all new files